lib/token-assert.js

```diff
@@ -31,19 +31,23 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
         if (nextToken.loc.start.line === token.loc.end.line &&
             (nextToken.loc.start.column - token.loc.end.column) !== spaces
         ) {
+            nextToken.whitespaceBefore = new Array(spaces + 1).join(' ');
```

Comment:
Anyway, we can defer this discussion till later, especially consider this is all consolidated to one spot considering we have this nice assertion layer.